ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -369,7 +369,7 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
         )
         progressEndCap.layer.cornerRadius = progressViewHeight/2
         progressEndCap.layer.masksToBounds = true
-        progressEndCap.backgroundColor = progressView.trackTintColor
+        progressEndCap.backgroundColor = quiz.currentIndex + 1 == questions.count ? progressView.progressTintColor : progressView.trackTintColor
```

Comment:
I like to think this line isn't needed. But depends on how the dev built the UI (so guess it is)